security/sec-iride/src/main/java/org/geoserver/security/iride/identity/IrideIdentity.java

```diff
@@ -111,12 +104,17 @@
     private transient String internalRepresentation;
 
     /**
+     * <code>IRIDE</code> Digital Identity entity formatter.
+     */
+    private final transient IrideIdentityFormatter formatter;

```

Comment:
Because I don't expect it to be serialized: a new formatter get created for every new IrideIdentity instance.